server/handler_test.go

```diff
@@ -264,6 +263,108 @@ func TestServer_SaveAndGetLayout(t *testing.T) {
 	})
 }
 
+func TestServer_SaveAndGetLayout_Dry(t *testing.T) {
+	workspaceId := fmt.Sprintf("workspace-%s", utils.RandString(8))
+	layoutId := fmt.Sprintf("layout-%s", utils.RandString(8))
+
+	jobQueue := dispatcher.NewInMemory()
+	dispatcher.Set(jobQueue)
+
+	plan := map[string]json.RawMessage{}
+
+	lBytes, err := ioutil.ReadFile("../runner/testdata/sleep.tf.json")
+	if err != nil {
+		t.Error(err)
+	}
+
+	plan["sleep.tf.json"] = uglyJson(lBytes)
+
+	vBytes, err := ioutil.ReadFile("../tmpl/testdata/vars.json")
+	if err != nil {
+		t.Error(err)
+	}
+
+	pBytes, _ := json.Marshal(plan)
+	vBytes = uglyJson(vBytes)
+
+	t.Run("Should create a layout in the workspace with dry flag", func(t *testing.T) {
+		req := &SaveLayoutRequest{Id: layoutId, WorkspaceId: workspaceId, Plan: pBytes, Dry: true}
+		resp, err := server.SaveLayout(context.Background(), req)
+
+		if err != nil {
+			t.Fatal(err)
+		}
+		newLayoutID := layoutId + drysuffix
+		assert.Equal(t, resp.LayoutId, newLayoutID)
```

Comment:
```suggestion
		assert.Equal(t, newLayoutID, resp.LayoutId)
```
`assert.Equal(t, <expected>, <got>)`